Assets/MixedRealityToolkit-Examples/StandardShader/Models/Materials.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
```

Comment:
Why do we have TWO materials folders in the new example.  Can the files in Models\Materials be transferred to the Top level materials folder in this example

Reply:
The Models\Materials folder is auto generated from the ShaderBall FBX, while the root Materials folder contains hand made materials. It's best to keep the auto generated materials in Models\Materials because any changes to the FBX will be automatically associated to existing materials in that folder.

Reply:
I happen to agree with @DDReaper here. We shouldn't auto import the materials.

Reply:
I can turn that settings off on the FBX, but I'm curious how do you handle potentially losing material references when FBX changes are made? Or, embedded media in the FBX? We've found our art team prefers working with the materials that get auto generated.

Reply:
I suppose it depends on the art team.  My initial experience with Unity was with 3d asset creation.

Generally, I don't like the default materials created by the editor, and usually end up changing them every single time, not to mention that sometimes the auto import will stomp any changes you made to the default material (although I suppose they could have fixed that by now).

Depending on the workflow, pipeline, and the technical abilities of the artists, some choose to write their own shaders in Maya/Max and export this way, and I believe that there may also be a way to correctly reference the shader to use from the modelling program itself on export. 

I've found that the only real important information from the model is the UVs, and managing the materials in the editor is generally the best route.

Reply:
I've consolidated the material folder and disabled auto import of materials. But, I would highly recommend giving the auto imported material workflow a second try if you get a chance! They work great for models which are being iterated on, plus any changes you make to the auto materials won't get stomped over.

Reply:
imo, models should only be delivered into a project when they're complete. haha.  Again, I suppose it's up to the processes and workflow of each team.  For the purposes of the open source project, we should deliver models as they're completed and not auto import the materials.